os/arch/arm/src/common/up_internal.h

```diff
@@ -278,7 +278,7 @@ EXTERN uint32_t _edata;			/* End+1 of .data */
 EXTERN uint32_t _sbss;			/* Start of .bss */
 EXTERN uint32_t _ebss;			/* End+1 of .bss */
 EXTERN uint32_t _sidle_stack;		/* Start of idle stack */
-EXTERN uint32_t __sint_heap_start;	/* Start of heap in internal RAM region */
+EXTERN uint32_t _sint_heap_start;	/* Start of heap in internal RAM region */
```

Comment:
Other than this change other file changes seem to be not related to the issue. Could you please check and revert those related changes?